docs/D-Modularity/functions.md

```diff
@@ -352,6 +369,8 @@ The structure of a walkthrough table for a modular program is a simple extension
 
 The completed walkthrough table for the power.c program listed above is shown below:
 
+* **& represents the address.**
+
 <table border="0">
 
```

Comment:
Add `<tbody>` for each table

Reply:
Thanks. I have fixed this and the other table issues 👍